gnome-initial-setup/gis-driver.c

```diff
@@ -98,6 +98,43 @@ gis_driver_finalize (GObject *object)
   G_OBJECT_CLASS (gis_driver_parent_class)->finalize (object);
 }
 
+static void
+prepare_main_window (GisDriver *driver)
+{
+  GisDriverPrivate *priv = gis_driver_get_instance_private (driver);
+  GdkGeometry size_hints;
+
+  if (gis_driver_is_small_screen (driver))
```

Comment:
The function does not take the driver anymore as an argument, so you should remove this from here (I'm surprised it builds without warnings!)